### PR TITLE
Complete NGSIV2 Unit tests

### PR DESCRIPTION
**Pull request by mapedraza**
Updating NGSI V2 Unit tests with cases included on NGSI V1 before deleting them

### CODE REVIEW

File: test/unit/ngsiv2/commandsAmqp-test.js [hunk 1]
```diff
@@ -136,6 +136,12 @@ describe('AMQP Transport binding: commands', function() {
                 done();
             });
         });
+        it('should reply with the appropriate command information', function(done) {
+            request(commandOptions, function(error, response, body) {
+                should.exist(body);
+                done();
+            });
+        });
```
Comment (mapedraza) [inline test/unit/ngsiv2/commandsAmqp-test.js:144]: Included tests from NGSI V1 version is failling in CI process

Comment (fgalan) [reply test/unit/ngsiv2/commandsAmqp-test.js:144]: I'll have a look to it.

**Comment by mapedraza** [comment]
MQTT topics have to be changed to /APIKEY/DEVICE-ID/attrs to /**json**/APIKEY/DEVICE-ID/attrs

File: test/unit/ngsiv2/commandsAmqp-test.js (hunk 1 above)
Comment (fgalan) [reply test/unit/ngsiv2/commandsAmqp-test.js:144]: I think I have realized why this works in NGSIv1 but not in NGSIv2.

In NGSIv1 all responses (even the ones for updates, like this one) has payload. With `should.exist(body)` you check that the response has a payload.

But in NGSIv2 you don't have response payload in the case of updates. Actually, the 204 No Content expectations check is asserting that behavior.

I suggest to remove this it() and incorporate the checking of no payload in the existing test above which should be something like this:

```
        it('should return a 204 OK without errors', function(done) {
            request(commandOptions, function(error, response, body) {
                should.not.exist(error);
                should.not.exist(body);
                response.statusCode.should.equal(204);
                done();
            });
        });
```

Comment (fgalan) [reply test/unit/ngsiv2/commandsAmqp-test.js]: Fixed in ee8be3c and ee8be3c

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**